firstvoices/backend/views/dictionary_views.py

```diff
@@ -92,7 +105,7 @@ class DictionaryViewSet(
     Dictionary entry information.
     """
 
-    http_method_names = ["get", "post", "put", "delete"]
+    http_method_names = ["get", "post", "put", "delete", "patch"]
```

Comment:
Probably better to remove this line and keep the default of allowing all methods (including HEAD etc)